build/testdata/bundles/wordpress/porter.yaml

```diff
@@ -42,13 +46,34 @@ install:
         externalDatabase.database: "{{ bundle.dependencies.mysql.parameters.database-name }}"
         #externalDatabase.host: "{{ bundle.dependencies.mysql.outputs.services }}"
         externalDatabase.user: "{{ bundle.dependencies.mysql.parameters.mysql-user }}"
-        externalDatabase.password: "{{ bundle.dependencies.mysql.outputs.mysql-password }}"
+        externalDatabase.password: "{{ bundle.parameters.mysql-password }}"
         externalDatabase.port: 3306
         mariadb.enabled: false
+      outputs:
+        - name: wordpress-password
+          secret: "{{ bundle.parameters.wordpress-name }}-wordpress"
+          key: wordpress-password
+        - name: mysql-password
+          secret: "{{ bundle.parameters.wordpress-name }}-externaldb"
+          key: db-password
 
 uninstall:
   - helm:
       description: "Uninstall Wordpress"
       purge: true
       releases: 
         - "{{ bundle.parameters.wordpress-name }}"
+
+outputs:
+  - name: wordpress-password
+    description: "The Wordpress installation password"
+    type: string
+    applyTo:
+      - "install"
+      - "upgrade"
+  - name: mysql-password
```

Comment:
This is an output of the mysql bundle, not the wordpress bundle. I'm really confused about why it's part of the wordpress bundle here.